src/shell/interpreter.zig

```diff
@@ -4933,25 +4938,8 @@ pub const Interpreter = struct {
                     .exit => "usage: exit [n]\n",
                     .true => "",
                     .false => "",
-                };
-            }
-
-            pub fn asString(this: Kind) []const u8 {
-                return switch (this) {
-                    .cat => "cat",
-                    .touch => "touch",
-                    .mkdir => "mkdir",
-                    .@"export" => "export",
-                    .cd => "cd",
-                    .echo => "echo",
-                    .pwd => "pwd",
-                    .which => "which",
-                    .rm => "rm",
-                    .mv => "mv",
-                    .ls => "ls",
-                    .exit => "exit",
-                    .true => "true",
-                    .false => "false",
+                    .yes => "yes [expletive]\n",
```

Comment:
heh, is that what it actually says? lol

Reply:
indeed 😄 

<img width="581" alt="image" src="https://github.com/oven-sh/bun/assets/5464072/fdd7f352-1363-4c31-a2f7-2805ccae6bce">